### ISSUE TITLE
Windows Linker warnings

### ISSUE THREAD

**Issue by rkwright** [readium/SDKLauncher-Windows#21]
When I build the Windows Launcher, I get a large number of warnings: A file with all the output is here:

https://readiumfoundation.box.com/s/r833s9hd2452efwx2ra604lc3yzn3inj

The zlib.pdb warnings aren't that serious.  It is an out of date zlib which needs to be upgraded to 64 bit per the work by Learning Ally.  I'm more concerned with the LNK4217 warnings, e.g. 

      CPP2JS.obj : warning LNK4217: locally defined symbol ?DurationMilliseconds_Calculated@SMILData@ePub3@@QBE?BIXZ etc.

I am suspicious of the fact that the EPUB code is generated as a static lib while Jim Dovey apparently expected it to be a DLL.  Note this code in _config.h

    #if EPUB_PLATFORM(WIN)
    # ifndef EPUB3_EXPORT
    #  ifdef BUILDING_EPUB3
    #   define EPUB3_EXPORT __declspec(dllexport)
    #  else
    #   define EPUB3_EXPORT __declspec(dllimport)
    #  endif
    # endif
    #else
    # define EPUB3_EXPORT
   #endif

This is classic Windows code for managing the code mapping between the DLL and the parent app.  I am going to experiment with converting the lib into a DLL.
The warnings do not seem to cause problems but perhaps might be related to the massive leaks. (see issue #15  )

**Comment by rkwright** [comment]
Update:  I added this code to _config.h:

    #undef EPUB3_EXPORT
    #define EPUB3_EXPORT

And the LNK4217 warnings all disappeared.  So that is the culprit, it appears. 
We still have two odd warnings:

    msvcrtd.lib(cinitexe.obj) : warning LNK4098: defaultlib 'msvcrt.lib' conflicts with use of other libs; use /NODEFAULTLIB:library
    ePub3.lib(node.obj) : warning LNK4049: locally defined symbol _xmlFree imported

I am compiling with /MDd on (use runtime libs as multi-threaded debug) so I suspect that one of the other libs being linked as a different threading model. 
The LNK4049 warning is worrisome, especially as the leaks seem to be from libxml.  Looking at that now.

**Comment by rkwright** [comment]
I have resolved the linker warnings, which were due to the fact that the SDK Core expected to be built as a DLL with the standard _declspec() macros defined.  However, the VS project for the core was building it as  a static lib and linking it to the Launcher as a static lib.  Since there was a conflict with the import statements, this generated a lot of warnings.  

I converted the SDK project to generate a DLL and changed the Launcher to load it as a DLL.  Warnings disappeared and it all works now.  Still leaks a LOT though.
